thirdeye-ui/package.json

```diff
@@ -64,7 +64,7 @@
         "@tanstack/react-query": "^4.29.19",
         "@tippyjs/react": "^4.2.6",
         "@visx/visx": "3.1.2",
-        "axios": "1.3.4",
+        "axios": "^1.7.4",
```

Comment:
use exact version without ^

Reply:
We are using the packacge.lock file and checking it out in source control too. Also we are using npm CI everywhere.
So we should be good with carat I think.